documentation/modules/appendix_crds.adoc

```diff
@@ -3012,11 +3012,15 @@ Used in: xref:type-KafkaMirrorMaker2Spec-{context}[`KafkaMirrorMaker2Spec`]
 |xref:type-KafkaMirrorMaker2ConnectorSpec-{context}[`KafkaMirrorMaker2ConnectorSpec`]
 |topicsPattern           1.2+<.<a|A regular expression matching the topics to be mirrored, for example, "topic1\|topic2\|topic3". Comma-separated lists are also supported.
 |string
-|topicsBlacklistPattern  1.2+<.<a|A regular expression matching the topics to exclude from mirroring. Comma-separated lists are also supported.
+|topicsBlacklistPattern  1.2+<.<a|*The `topicsBlacklistPattern` property has been deprecated, and should now be configured using `topicsExcludePattern`.* A regular expression matching the topics to exclude from mirroring. Comma-separated lists are also supported.
```

Comment:
You will need to rebuild the API documentation.

Reply:
@laidan6000 I think these descriptions are auto generated from the api module.

Reply:
@laidan6000 I think these descriptions are auto generated from the api module.

Reply:
TBH, I think we should keep it as it is. This is the generated default for all the fields we have.

Reply:
But I thought all descriptions could be changed by editing the Java files in the `api/` folder?

Reply:
@laidan6000 I could remove the property description from deprecated properties, but changing the deprecation message would be out of scope for this PR as it would affect all deprecated properties.

Reply:
But why should the deprecated properties have no description? Users will be still using them so I think keeping the description makes sense ... that is also what we did for all other deprecated properties in the past.

Reply:
Yep, I agree @scholzj.

Reply:
Thanks for the explanation. I didn't realise that this would affect all the deprecated properties.